addons/sale/tests/test_reinvoice.py

```diff
@@ -115,6 +116,61 @@ def test_at_cost(self):
         self.assertEqual((sale_order_line6.price_unit, sale_order_line6.qty_delivered, sale_order_line6.product_uom_qty, sale_order_line6.qty_invoiced), (self.company_data['product_delivery_cost'].standard_price, 2, 0, 0), 'Sale line 6 is wrong after confirming 2e vendor invoice')
 
+    def test_sales_team_invoiced(self):
+        """ Test invoiced field from  sales team ony take into account the amount the sales channel has invoiced this month """
+
+        bank_journal = self.env['account.journal'].create({'name': 'Bank', 'type': 'bank', 'code': 'BNK67'})
+        move_form = Form(self.env['account.move'].with_context(default_move_type='out_invoice'))
+        move_form.partner_id = self.partner_a
+        move_form.invoice_date = fields.Date.today()
+        move_form.date = fields.Date.today()
+        with move_form.line_ids.new() as line_form:
+            line_form.product_id = self.company_data['product_order_cost']
+            line_form.quantity = 1.0
+        out_invoice = move_form.save()
+        out_invoice.action_post()
+
+        payment = self.env['account.payment'].create({
+            'payment_type': 'inbound',
+            'payment_method_id': self.env.ref('account.account_payment_method_manual_in').id,
+            'partner_type': 'customer',
+            'partner_id': self.partner_a.id,
+            'amount': out_invoice.amount_total,
+            'journal_id': bank_journal.id,
+            'date': fields.Date.today(),
+        })
+        payment.action_post()
+
+        credit_aml = payment.line_ids.filtered('credit')
+        out_invoice.js_assign_outstanding_line(credit_aml.id)
+
+        move_form = Form(self.env['account.move'].with_context(default_move_type='in_invoice'))
+        move_form.partner_id = self.partner_a
+        move_form.invoice_date = fields.Date.today()
+        move_form.date = fields.Date.today()
+        with move_form.line_ids.new() as line_form:
+            line_form.product_id = self.company_data['product_order_cost']
+            line_form.quantity = 1.0
+        in_invoice = move_form.save()
+        in_invoice.action_post()
+
+        payment = self.env['account.payment'].create({
+            'payment_type': 'outbound',
+            'payment_method_id': self.env.ref('account.account_payment_method_manual_in').id,
+            'partner_type': 'supplier',
+            'partner_id': self.partner_a.id,
+            'amount': in_invoice.amount_total,
+            'journal_id': bank_journal.id,
+            'date': fields.Date.today(),
+        })
+        payment.action_post()
+
+        debit_aml = payment.line_ids.filtered('debit')
+        in_invoice.js_assign_outstanding_line(debit_aml.id)
+
+        self.assertEqual(move_form.team_id.invoiced, 280, "The amount the sales channel has invoiced this month is 280€")
```

Comment:
always use `assertRecordValues`, specially if you are comparing monetary values...